src/Sylius/Behat/Page/Shop/Checkout/AddressPage.php

```diff
@@ -90,12 +81,7 @@ public function checkInvalidCredentialsValidation()
         return $this->checkValidationMessageFor('login_password', 'Invalid credentials.');
     }
 
-    /**
-     * {@inheritdoc}
-     *
-     * @throws ElementNotFoundException
-     */
-    public function checkValidationMessageFor($element, $message)
+    public function checkValidationMessageFor(string $element, string $message): bool
```

Comment:
in line 88, you should change validation error from class do data-attributes like this https://github.com/Sylius/Sylius/pull/10883/files#diff-22c371129177dc765960e9e7aa87a410R31

Reply:
here and bottom: on this PR I only added type hints and deleted unnecessary doc blocks